content/blog/2016/2016-05-04-the-state-of-JAM

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "The State of Jenkins Area Meetup (JAM)"
+tags:
+- meetup
+- JAM
+- event
+author: alyssat
+---
+
+NOTE: _This is a guest post by link:https://github.com/alyssat[Alyssa Tong].
```

Comment:
@alyssat is the Events Officer, since this is pertaining to her duties as such, I don't think "guest post" makes sense here.

For the record, I have also privately given her sufficient grief about using a ghost writer for her blog posts